### PR TITLE
Button up the design and front-end

### PR DESCRIPTION
**Pull request by tysongach**
## Before

localhost-4567- laptop with mdpi screen 1

localhost-4567- iphone 6

## After

localhost-4567- laptop with mdpi screen

localhost-4567- iphone 6 1

### CODE REVIEW

File: source/stylesheets/base/_layout.scss [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+html,
+body {
+  height: 100%;
```
Comment (kylefiedler) [inline source/stylesheets/base/_layout.scss]: do we need this?

File: source/stylesheets/library/_variables.scss [hunk 2]
```diff
@@ -0,0 +1,43 @@
+// Typography
+$base-font-family: "freight-text-pro", $font-stack-georgia;
```
Comment (kylefiedler) [inline source/stylesheets/library/_variables.scss]: Since we're not using Freight in our brand anymore and this is departed from thoughtbot branding anyway, thoughts on just leaving these to both the system stack?

File: source/stylesheets/components/_footer.scss [hunk 3]
```diff
@@ -1,10 +1,6 @@
-$desktop-small: new-breakpoint(max-width 1024px);
-$mobile: new-breakpoint(max-width 480px);
-$iPhone5-min-breakpoint: 320px;
-$iPhone5-max-breakpoint: 568px;
-
 .site-footer {
   background-color: $thoughtbot-gray;
```
Comment (kylefiedler) [inline source/stylesheets/components/_footer.scss]: I say we ditch this footer color. Might be another PR for that though?

File: source/stylesheets/components/_footer.scss [hunk 4]
```diff
@@ -1,10 +1,6 @@
...
+  margin-top: $base-spacing * 2;
```
Comment (kylefiedler) [inline source/stylesheets/components/_footer.scss]: Against doing `$large-spacing`?

**Comment by kylefiedler** [comment]
Sorta kills my soul that we indent the bullet lists

File: source/stylesheets/library/_variables.scss (hunk 2 above)
Comment (tysongach) [inline source/stylesheets/library/_variables.scss]: Good idea. I’m game for that.

File: source/stylesheets/base/_layout.scss (hunk 1 above)
Comment (tysongach) [inline source/stylesheets/base/_layout.scss]: Nope! ✂️

File: source/stylesheets/components/_footer.scss (hunk 3 above)
Comment (tysongach) [inline source/stylesheets/components/_footer.scss]: Yeah, it’s a tad strong. I’ll add a commit here for that, since we are buttoning up the whole design a bit.

File: source/stylesheets/components/_footer.scss (hunk 4 above)
Comment (tysongach) [inline source/stylesheets/components/_footer.scss]: No, I probably should. Good catch!

**Comment by tysongach** [comment]
> Sorta kills my soul that we indent the bullet lists

You prefer hanging bullets?

**Comment by kylefiedler** [comment]
> You prefer hanging bullets?

For long prose like this, yes.

**Comment by tysongach** [comment]
Updated. See Netlify deploy preview: https://deploy-preview-31--apprentice.netlify.com/

localhost-4567- laptop with hidpi screen

localhost-4567- iphone 6 1

**Comment by tysongach** [comment]
Yay, one less Typekit kit.

**Comment by kylefiedler** [comment]
This looks good. I don't know how much you want to get into the design. Other suggestions design-wise:

- Tighten the spacing between main title and subtitle 
- More spacing above titles
- Bump the type on bigger screens

**Comment by tysongach** [comment]
@kylefiedler Tweaked: https://deploy-preview-31--apprentice.netlify.com/

**Comment by kylefiedler** [comment]
🚢  ⛵️ 💯 🐑
